_pytest/pytester.py

```diff
@@ -908,8 +908,11 @@ def popen(self, cmdargs, stdout, stderr, **kw):
         env['PYTHONPATH'] = os.pathsep.join(filter(None, [
             str(os.getcwd()), env.get('PYTHONPATH', '')]))
         kw['env'] = env
-        return subprocess.Popen(cmdargs,
-                                stdout=stdout, stderr=stderr, **kw)
+
+        popen = subprocess.Popen(cmdargs, stdin=subprocess.PIPE, stdout=stdout, stderr=stderr, **kw)
```

Comment:
Oh, you are right. The original solution was `subprocess.DEVNULL` as `stdin`, but this was not available in older python version. Therefore I replaced it with a closed pipe, but I forgot to change the commit message too.